app/assets/javascripts/application.js

```diff
@@ -12,6 +12,8 @@
 //
 //= require jquery
 //= require jquery_ujs
+//= require dataTables/jquery.dataTables
```

Comment:
why is this necessary given https://github.com/projecthydra-labs/lerna/pull/335/files ?